Source/Core/DolphinQt/NetPlay/NetPlayDialog.cpp

```diff
@@ -292,6 +298,24 @@ void NetPlayDialog::ConnectWidgets()
           [this] { m_chat_send_button->setEnabled(!m_chat_type_edit->text().isEmpty()); });
 
   // Other
+  connect(m_auto_buffer_box, &QCheckBox::clicked, this, [this](bool value) {
+    m_buffer_size_box->setEnabled(!value);
+    m_buffer_label->setEnabled(!value);
+
+    if (value)
+    {
+      auto server = Settings::Instance().GetNetPlayServer();
+      if (server && !m_host_input_authority)
+      {
+        const u32 pad_buffer = server->CalculatePadBuffer();
+
+        Config::SetBase(Config::NETPLAY_BUFFER_SIZE, pad_buffer);
+        server->AdjustPadBufferSize(pad_buffer);
+        m_buffer_size_box->setValue(pad_buffer);
```

Comment:
I'm pretty sure you can remove the `SetBase` and `AdjustPadBufferSize` calls, as `setValue` should fire the `valueChanged` signal that already does the right thing.